nat-lab/tests/test_mesh_exit_through_peer.py

```diff
@@ -91,13 +91,18 @@ async def test_mesh_exit_through_peer(
         )
 
         client_alpha = await exit_stack.enter_async_context(
-            telio.Client(connection_alpha, alpha, adapter_type).run_meshnet(
-                api.get_meshmap(alpha.id)
-            )
+            telio.Client(
```

Comment:
Do we really need these reformattings?

Reply:
Just my python formatter just did it. I can undo them.

Reply:
:thinking: probably would be nice to minimze the `diff`, but this is more of a nitpick kind of thing, so up to you to decide ;)